src/overlays/actors/ovl_Arrow_Fire/z_arrow_fire.c

```diff
@@ -68,7 +68,7 @@ void ArrowFire_Charge(ArrowFire* this, PlayState* play) {
     }
 
     if (this->radius < 10) {
-        this->radius += 1;
+        this->radius++;
```

Comment:
imo this shouldn't be ++, it may as well be `+= 2` if the radius should expand quicker, just happens to be 1 every frame

same with arrow_ice and arrow_light

Reply:
Apologies for the delay responding to these. My small brain still doesn't interpret `++` to be different than `+= 1` so these comments seems confusing to me. But yeah, made all the suggestions here